python/lsst/sims/maf/metrics/baseMetric.py

```diff
@@ -86,9 +86,9 @@ class BaseMetric(object):
     colRegistry = ColRegistry()
     colInfo = ColInfo()
 
-    def __init__(self, col=None, metricName=None, maps=None, units=None,
+    def __init__(self, col='metricdata', metricName=None, maps=None, units=None,
                  metricDtype=None, badval=-666,
-                 plotDict=None, displayDict=None):
+                 plotDict=None, displayDict=None, summaryStatList=None):
```

Comment:
Okay - I went and talked to Peter and we came up with a PLAN. We'll make a new branch to implement our new """BENCHMARK PLAN""".